packages/codec/lib/format/index.ts

```diff
@@ -37,8 +37,8 @@
  * types.
  *
  * This format targets types and values understood by the
- * [Solidity programming language](https://solidity.readthedocs.io) and
- * the [Contract ABI specification](https://solidity.readthedocs.io/en/v0.5.3/abi-spec.html),
+ * [Solidity programming language](https://docs.soliditylang.org/) and
+ * the [Contract ABI specification](https://docs.soliditylang.org/en/v0.8.17/abi-spec.html),
```

Comment:
Of course you probably now want to update this to .19 :)

Reply:
Ha, I could!